src/Util/Uuid.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * This file is part of `prooph/event-store-client`.
+ * (c) 2018-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2018-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStoreClient\Util;
+
+use Ramsey\Uuid\FeatureSet;
+use Ramsey\Uuid\UuidFactory;
+use Ramsey\Uuid\UuidInterface;
+
+class Uuid
+{
+    /** @var UuidFactory */
+    private static $factory;
+
+    public static function generate(): UuidInterface
```

Comment:
Shouldn't this and other methods returning UuidInterface be private? This way there are still some ramsey/uuid instances used elsewhere.

Reply:
Not sure at the moment. I still use it in `EventId::generate()`

Reply:
Yeah I noticed. What is the reason?

Reply:
On the `EventId` I need the methods `toString()` and `toBinary()`. I could work around and always recreate the uuid object from a string to then turn it into binary, but that's some overhead.

I'm open for suggestions or PR's though.

Reply:
Thanks for explanation. I think this is good to merge now, we can revisit these details later.